src/styles/blocks/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid $gray-border;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    @include font($size: 12px, $weight: 500);
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__description {
+    align-self: start;
+    margin-bottom: 16px;
+    line-height: 14px;
+    @include font($size: 10px);
+    color: $secondary-accent;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 24px;
+    align-items: center;
+
+    &-count {
+      @include font($size: 10px);
+      color: $main-accent;
+      line-height: 16px;
+    }
+  }
+
+  .card__price {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 16px;
+    align-items: center;
+
+    &-text {
+      @include font($size: 12px);
+      color: $secondary-accent;
+    }
+    &-sum {
+      @include font($size: 16px, $weight: 700);
+      line-height: 18px;
+      color: $main-accent;
+    }
+  }
+
+  &__button {
+    padding: 11px 0;
```

Comment:
you failed tests because of this
add 
`    height: 40px;
    display: flex;
    align-items: center;
    justify-content: center;`
instead of padding